src/pages/git/git-rm/index.md

```diff
@@ -0,0 +1,23 @@
+## Git rm
+
+The ```git rm``` command allows you to remove files from the working tree and from the index.
```

Comment:
You can change this to `git rm` with a single back-tick instead of 3 on either side